src/linear_algebra/dual.lean

```diff
@@ -529,6 +529,15 @@ end
 
 variables [finite_dimensional K V] [finite_dimensional K V₁]
 
+lemma findim_eq_dual :
+  findim K V = findim K (module.dual K V) :=
```

Comment:
This probably makes more sense the other way around, and is maybe even sensible as a `simp` lemma